action/protocol/execution/testdata/datacopy.json

```diff
@@ -0,0 +1,28 @@
+{
+    "initBalances": [{
+        "account": "io1mflp9m6hcgm2qcghchsdqj3z3eccrnekx9p0ms",
+        "rawBalance": "1000000000000000000000000000"
+    }],
+    "deployments": [{
+        "rawByteCode": "608060405234801561001057600080fd5b506102b9806100206000396000f3fe608060405234801561001057600080fd5b506004361061002b5760003560e01c8063574c807814610030575b600080fd5b61003861004e565b60405161004591906101fe565b60405180910390f35b600080600367ffffffffffffffff81111561006c5761006b610254565b5b6040519080825280601f01601f19166020018201604052801561009e5781602001600182028036833780820191505090505b509050601160f81b816000815181106100ba576100b9610225565b5b60200101907effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916908160001a905350602260f81b8160018151811061010257610101610225565b5b60200101907effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916908160001a905350600060f81b8160028151811061014a57610149610225565b5b60200101907effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916908160001a90535060008151905060008167ffffffffffffffff81111561019b5761019a610254565b5b6040519080825280601f01601f1916602001820160405280156101cd5781602001600182028036833780820191505090505b509050600082602185018460208701600462010000fa90508094505050505090565b6101f881610219565b82525050565b600060208201905061021360008301846101ef565b92915050565b60008115159050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fdfea2646970667358221220aa7f630a0a650f0b72641e0fcfc51caaa403d31e1a7a29f54af3529136235f3964736f6c63430008070033",
+        "rawPrivateKey": "cfa6ef757dee2e50351620dca002d32b9c090cfda55fb81f37f1d26b273743f1",
+        "rawAmount": "0",
+        "rawGasLimit": 5000000,
+        "rawGasPrice": "0",
+        "rawExpectedGasConsumed": 222483,
+        "expectedBalances": [],
+        "comment": "deploy datacopy contract"
+    }],
+    "executions": [{
+        "readOnly": true,
+        "rawPrivateKey": "cfa6ef757dee2e50351620dca002d32b9c090cfda55fb81f37f1d26b273743f1",
+        "rawByteCode": "574c8078",
```

Comment:
can we construct a raw bytes similar to attack tx: https://tx.blocksecteam.com/tx/0x1cb6fb36633d270edefc04d048145b4298e67b8aa82a9e5ec4aa1435dd770ce4
and play the tx on testnet?